samples/msal-node-samples/standalone-samples/ElectronTestApp/src/AuthProvider.ts

```diff
@@ -135,12 +135,16 @@ export default class AuthProvider {
     }
 
     async loginSilent(): Promise<AccountInfo> {
-        return this.account || await this.getAccount();
+        if (!this.account) {
+            this.account = await this.getAccount();
+        }
+
+        return this.account;
     }
 
     async logout(): Promise<void> {
         if (this.account) {
-            await this.clientApplication.getTokenCache().removeAccount(this.account);
+            this.clientApplication.getTokenCache().removeAccount(this.account);
```

Comment:
Why is this no longer awaited?

Reply:
Thanks for catching it, I undid changes and forgot to put it back. I'll put it back.